wordpress-mobile/WordPress-Android
# My Site Dashboard: Adjust QS focus points for Stats and Pages

Parent #16301

This PR removes focus points for `Stats` and `Pages` from `Quick Links Card`, `Quick Links Ribbon`, and displays them on rows (similar to iOS).
Internal Ref: p1649763254844489/1649694944.738209-slack-C0290FLA0RM

/cc @tiagomar 

To test:

#### Test: QS Task: Edit your homepage

Prerequisite: Initial Tab: Home

1. Launch the app.
2. Select `Customize your site` in the `Next Steps` card.
3. Select `Edit your homepage`.
4. ❌ Notice that QS focus point is not shown on `Quick Links` -> `Pages`.
5. ✅ Notice that "Tap Menu to Continue" snackbar is shown with focus point on `Menu` tab.
6. Click `Menu` tab.
7. ✅ Notice that 
    - `Select pages to see your list` snackbar is shown
    - QS focus point is shown on `Pages` site item 
9. Click `Pages` list item.
10. ✅ Notice that QS focus point and snackbar is shown on the `Homepage` list item on `Pages` screen.

Repeat above steps for `Initial Tab: Menu` test skipping steps 5, 6 and confirm that focus points are displayed correctly on Stats and Pages list items.

#### Test: QS Task: Review site pages

Prerequisite: Initial Tab: Home

1. Launch the app.
2. Select `Customize your site` in the `Next Steps` card.
3. Select `Review site pages`.
4. ❌  Notice that QS focus point is not shown on `Quick Links` -> `Pages`.
5. ✅ Notice that "Tap Menu to Continue" snackbar is shown with focus point on `Menu` tab.
6. Click `Menu` tab.
7. ✅ Notice that 
    - `Select pages to see your list` snackbar is shown
    - QS focus point is shown on `Pages` site item 
9. Click `Pages` list item.
10. ✅ Notice that `Pages` screen is opened.

Repeat above steps for `Initial Tab: Menu` test skipping steps 5, 6 and confirm that focus points are displayed correctly on Stats and Pages list items.

#### Test: QS Task: Check your stats

Prerequisite: Initial Tab: Home

1. Launch the app.
2. Select `Grow your audience` in the `Next Steps` card.
3. Select `Check your stats.
4. ❌ Notice that QS focus point is not shown on `Quick Links` -> `Stats`.
5. ✅ Notice that "Tap Menu to Continue" snackbar is shown with focus point on `Menu` tab.
6. Click `Menu` tab.
7. ✅ Notice that 
    - `Tap Stats to see how your stats are performing.` snackbar is shown.
    - QS focus point is shown on `Stats` list item. 
9. Click `Stats` list item.
10. ✅ Notice that `Stats` screen is opened.

Repeat above steps for `Initial Tab: Menu` test skipping steps 5, 6 and confirm that focus points are displayed correctly on Stats and Pages list items.

#### Test: Without Tabs

- Disable the tabs feature from Debug Settings and restart the app.
- Repeat the above tests and confirm that focus points are displayed correctly on Stats and Pages list items. Note that `Site Menu Step` (step 5, 6 in tests) will not be shown when tabs are disabled.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
